python_operator/template.py

```diff
@@ -146,6 +147,33 @@ def _add_submodule(self, output_dir):
                 "the Operator framework as a submodule:\n{}".format(str(e))
             )
 
+    def _git_commit(self, output_dir, msg="Initial commit"):
```

Comment:
I think this should be something more generic like "Automatic commit by `charm create` using the python operator template"

Reply:
Since this just the generic commit function, what if we lave this as a very standard initial commit message, but update the usage in charm-template to make it a more specific message.